pkg/sfu/downtrack.go

```diff
@@ -131,7 +132,12 @@ func NewDownTrack(c webrtc.RTPCodecCapability, r Receiver, bf *buffer.Factory, p
 		receiver:      r,
 		codec:         c,
 		munger:        NewMunger(),
-	}, nil
+	}
+	if strings.ToLower(c.MimeType) == "video/vp8" {
```

Comment:
Creating a munger here every time. This should take care of the multi-threading issue you mentioned @davidzhao . It is a bit of overkill for simple tracks without temporal layers as this is really needed only when sending the last blank frames, but should not be a big burden. But, something to keep an eye on.